awpy/plot/utils.py

```diff
@@ -34,10 +34,40 @@ def position_transform_axis(
     return (start - position) / scale
 
 
+def position_revert_axis(
```

Comment:
What's the intended use for this function?

Reply:
Given a point that is plotted on an image, users can feed the pixel co-ordinates of that point into `position_revert_axis()` and get the in-game co-ordinates of that point.

I am using your library to plot points on a matplotlib graph, draw a polygon over it, and return my polygon to a database in terms of in-game coordinates.

Reply:
I see. Perhaps what we can do is rename the function. Usually for these transformer-style function we can name them like "X_to_Y". We probably should rename the `position_transform()` function, we can still do that, I'm all ears for names!

Reply:
I agree; I would name the functions `in-game_to_pixel_transform` and `pixel_to_in-game_transform`, but I didn't take the liberty due to - again - backwards compatibility reasons.

Maybe rename the functions to `in-game_to_pixel_transform` & `pixel_to_in-game_transform` and create "new" functions  that go by the old names (`position_transform_axis` & `position_revert_axis`) that simply call `in-game_to_pixel_transform` & `pixel_to_in-game_transform` and send a deprecation warning in the console?

Reply:
I think we can do the following:

game_to_pixel(...)
pixel_to_game(...)

since "in game" is redundant with "game" and "transform" is implied by the "to"

then, the interface is much cleaner and imo, easy to tell what the functions do